src/pages/api/teams/create.ts

```diff
@@ -0,0 +1,101 @@
+import ShortUniqueId from "short-unique-id";
+import { createClient } from "@supabase/supabase-js";
+import { sendEmail } from "~/pages/api/utils/mailer.ts";
+import type { APIRoute } from "astro";
+import type { CreateTeamItem } from "~/types";
+
+export const prerender = false;
+
+const supabase = createClient(
+  import.meta.env.SUPABASE_URL,
+  import.meta.env.SUPABASE_ANON_KEY,
+);
```

Comment:
Not properly related to this PR, but shouldn't we have only one client throughout the whole application?

Reply:
Yes we should. I exported the email client to another module, for example. I'll do that in the future 💪